src/components/IOUBadge.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Text, TouchableOpacity} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import Num from 'expensify-common/lib/Num';
+import ONYXKEYS from '../ONYXKEYS';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // IOU Report data object
+    iouReport: {
```

Comment:
I think this is missing `PropType shape`

Reply:
`iouReport: PropTypes.shape({...})`

Reply:
Ah yep I rushed this one 😅